modules/cluster-and-ha-management/pages/ha-overview.adoc

```diff
@@ -0,0 +1,70 @@
+* New query installation will only resume once a node is back online; however, interpreted queries will continue as before.
+* A schema changes will return an error until the node is back online.
+* Backup and export operations continue as normal once failed nodes are back online or removed.
+
+NOTE: Workarounds to the above instances can be archived by the xref:_removal_of_failed_nodes[removal of the failing nodes].
+
+== High Availability Cluster Configuration
+
+High Availability (HA) xref:tigergraph-server:cluster-and-ha-management:ha-cluster.adoc[Cluster Configuration] provides a resilient framework with flexibility for resizing and optimizing data distribution in TigerGraph systems.
+
+Key factors include:
+
+* The replication factor.
+* The partitioning factor.
+* The cluster size.
+
+== High Availability Support for GSQL Server
+
+High Availability (HA) support for xref:tigergraph-server:cluster-and-ha-management:ha-for-gsql-server.adoc[GSQL Server] ensures robustness by allowing components to take over in case of unavailability.
+Incorporating a primary GSQL server and standby servers, collectively referred to as leaders and followers, offering HA for client connections.
+
+NOTE: It is recommended to couple this with a client-side retry logic, so failing user query will complete once the system adjusts to accommodate the failed node (typically up to 30 seconds).
+
+== High Availability Support for Application Server
+
+TigerGraph incorporates native High Availability (HA) support for its xref:tigergraph-server:cluster-and-ha-management:ha-for-application-server.adoc[application server], ensuring uninterrupted service for GraphStudio and Admin Portal APIs.
+The active-active architecture runs on the first three nodes by default, in case of server failure, users can switch to an available server within the cluster.
+
+NOTE: If the primary node is offline, access to GraphStudio is interrupted, but resumes once the primary node is back online.
+
+In the event of a server failure, users can transition to the next available server within the cluster to resume operations,
+with a reminder that any ongoing long-running operations will be lost.
+
+.Additionally, load balancing supports the following services:
+* Nginx
+* AWS Elastic Load Balancer
+* Azure Application Gateway
+* GCP External HTTP(s) Load Balancer
+
+== Cluster Commands
+
+xref:tigergraph-server:cluster-and-ha-management:cluster-commands.adoc[Advanced Linux commands] that simplify platform operations can performed by the platform owner during debugging on HA clusters.
+
+== Removal of Failed Nodes
+
+HA cluster configuration allows for the xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[removal of failed nodes] without data loss, facilitating cluster maintenance and recovery.
+This feature is especially useful when addressing individual node failures or handling multiple concurrent node failures in a prepared environment, covering:
```

Comment:
Not here but on the Removal of Failed Nodes page, I think we can improve the explanation with an illustrative example.